goldenrun.py

```diff
@@ -294,6 +294,11 @@ def generate_wildcard_faults(fault, tbexec, tbinfo):
                         }
                     )
 
+            # Evaluate exclude ranges
```

Comment:
I checked again, you can move this to line 265, so in front of the "Analyze TB" block. This may result in a slight performance advantage in some cases.